pkg/ibm/ibmp.go

```diff
@@ -155,10 +155,7 @@ func (r IBMPowerDynamicConfig) ListInstances(kubeClient client.Client, ctx conte
 			continue
 		}
 		identifier := cloud.InstanceIdentifier(*instance.PvmInstanceID)
-		createdAt, err := time.Parse(time.RFC3339Nano, instance.CreationDate.String())
-		if err != nil {
-			log.Error(err, "cannot parse creation time", "instance", identifier)
-		}
+		createdAt := time.Time(instance.CreationDate)
```

Comment:
is this a type cast? why do we need it? What is the type on CreationDate?

Reply:
it is [strfmt.DateTime](https://github.com/go-openapi/strfmt/blob/master/time.go#L125) which is json utility wrapper around `time.Time`

Reply:
ok, thanks.